modules/msg/fsw/src/cfe_msg_ccsdsext.c

```diff
@@ -90,7 +90,7 @@ CFE_Status_t CFE_MSG_GetEDSVersion(const CFE_MSG_Message_t *MsgPtr, CFE_MSG_EDSV
  *-----------------------------------------------------------------*/
 CFE_Status_t CFE_MSG_SetEDSVersion(CFE_MSG_Message_t *MsgPtr, CFE_MSG_EDSVersion_t Version)
 {
-    if (MsgPtr == NULL || (Version > (CFE_MSG_EDSVER_MASK >> CFE_MSG_EDSVER_SHIFT)))
+    if (MsgPtr == NULL || Version == NULL || (Version > (CFE_MSG_EDSVER_MASK >> CFE_MSG_EDSVER_SHIFT)))
```

Comment:
Version can be 0...

Reply:
> Good work! See requested changes... those values can all be zero (they aren't pointers).

Thanks...

Sure, I'll change that back quickly.